scripts/values.sh

```diff
@@ -1,26 +1,13 @@
 #! /bin/bash
```

Comment:
Do we still use this script? I can't remember what it was used for

Reply:
It's actually used as part of make images.  We could scrap it though.  Not sure how much value it provides